shell.nix

```diff
@@ -35,7 +36,9 @@
   venvDir = "./.venv";
 
   postShellHook = ''
-    pip install --upgrade pip
     pip install -r ${./contracts/requirements.txt} -c ${./contracts/constraints.txt}
+    helm repo add chainlink-qa https://raw.githubusercontent.com/smartcontractkit/qa-charts/gh-pages/
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    helm repo update
```

Comment:
An idea: add the `~/.config/helm/repositories.yaml` file to the project, then set `HELM_REPOSITORY_CONFIG` in this `shell.nix` to point to the file. That way it's part of the project and doesn't require running shell hooks